### ISSUE TITLE
Signup link from Login Form doesn't work 

### ISSUE THREAD

**Issue by xandro0777** [woheller69/huggingassist#13]
.. I get just a bright white screen.

**Comment by woheller69** [comment]
Please sign up with another browser. I do not want to allow Amazon Web Application Firewall url

**Comment by xandro0777** [comment]
Ok.. could there be in the menu an option to open the signup page in a browser? Or could the signup be caught and redirected to an external app?

**Comment by woheller69** [comment]
you only need to to this once in your life. Login with your credentials works

**Comment by xandro0777** [comment]
Yes.. would be still nice to have the link in some obvious place
